### PR TITLE
PMM-8267: backup status loading animation

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Adds a loading animation for backups and restores

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-8267

https://user-images.githubusercontent.com/4190654/128165751-534ef06e-7bf1-46be-b460-50227567d02f.mov

### CODE REVIEW

File: public/app/percona/backup/components/Status/Status.styles.ts [hunk 1]
```diff
@@ -13,14 +13,20 @@ export const getStyles = stylesFactory((theme: GrafanaTheme, status: BackupStatu
...
+  const result = {
+    ellipsisContainer: css`
+      display: table;
+      width: 20px;
+      margin: 0 auto;
+    `,
+    status: '',
+  };
 
   if (isSuccess || isError) {
```
Comment (tiagomotasantos) [inline public/app/percona/backup/components/Status/Status.styles.ts:25]: I think you could benefit from conditional styles with the `cx` function so the styles file would be just the styles and  the conditions/logic would be on the component.

Like here: https://github.com/percona-platform/grafana/blob/7dffd14498dbd9900ab5f596c71364be6786a8a3/public/app/percona/dbaas/components/DBCluster/DBClusterStatus/DBClusterStatus.tsx#L20

Just a suggestion anyway.

File: public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { css, keyframes } from 'emotion';
+
+const bounce = keyframes`
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts:3]: Could you please share a video/GIF of how this looks?

**Review by tiagomotasantos — APPROVED** [review]

File: public/app/percona/backup/components/Status/Status.styles.ts (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/backup/components/Status/Status.styles.ts]: Thanks, way cleaner !

File: public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts (hunk 2 above)
Comment (fabio-silva) [reply public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts:3]: Sure, done 👍

Comment (tiagomotasantos) [reply public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts:3]: Great stuff! Will probably use it in DBaaS!

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**